plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/IcebergQueryRunner.java

```diff
@@ -479,13 +479,20 @@ private IcebergQueryRunnerWithTaskRetries() {}
         public static void main(String[] args)
                 throws Exception
         {
-            Path exchangeManagerDirectory = createTempDirectory(null);
+            Logger log = Logger.get(IcebergQueryRunnerWithTaskRetries.class);
```

Comment:
Does it matter whether the logger is created here or as it was before ?

Reply:
It does not - just consistent with other classes.